numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -1077,8 +1077,9 @@ def add_newdoc(place, name, doc):
 
     Returns
     -------
-    out : ndarray or bool
-        Output array of bools.
+    out : ndarray or scalar
+        Output array, element-wise comparison of `x1` and `x2`.
+        Typically of type bool, unless ``dtype=object`` is passed.
```

Comment:
A nitpick, but why `Typically`? Just remove it if it is *always* type bool unless specified otherwise.

Reply:
A custom-registered ufunc loop could return some other type too.

Reply:
But this is the docstring for the standard functions; I'm not sure if it... If we're going even more nitpicky, the `dtype` of any passed-in `out` argument is of course preserved. Anyway, it is also fine to leave as is...

Reply:
I think I'd like to leave it as is, if that's alright

Reply:
OK, no problem.